views/matrixsplit.html

```diff
@@ -497,18 +496,17 @@
             \end{bmatrix}
             \cdot
             \begin{bmatrix}
-              z^2 \cdot P_3 - 2 \cdot z \cdot (z-1) \cdot P_2 + (z-1)^2 \cdot P_1 \\
-              z \cdot P_3  - (z-1) \cdot P_2 \\
-              P_3
+              (1-z)^2 \cdot P_1 + 2 \cdot (1-z) \cdot z \cdot P_2 + z^2 \cdot P_3  \\
+                                  (1-z) \cdot P_2                 + z \cdot P_3 \\
```

Comment:
For this and all the others: provided that HTML/MathJax and LaTeX are not annoyed by the spaces, I found it useful to do the indenting for double checking that I was including all the relevant terms (it was easier to compare against the matrices). I can change the pull request to eliminate them of course.